debezium-core/src/main/java/io/debezium/transforms/debezium-timestamp-converter/src/main/java/org/telmate/SMT/OptionalTypeFieldSchema.java

```diff
@@ -0,0 +1,13 @@
+package org.telmate.SMT;
```

Comment:
All files should have the standard header, see one of the existing ones for the header.